README.md

```diff
@@ -315,6 +316,8 @@ For `OnPlayChoiceEvent` you must also pass an integer representing the index of
 selected then you should pass `INDEX_NONE` as the index. Note that this will result in no choice statements being executed - the Supertalk player will simply
 continue to the next statement after the current set of choices.
 
+After you've created the Supertalk player, you can run `USupertalkPlayer::RunScript(USupertalkScript* Script)`.
```

Comment:
"After you've created the Supertalk player, you can run scripts with `USupertalkPlayer::RunScript(USupertalkScript* Script)`."